cells/lib/ops/mkOCI.nix

```diff
@@ -70,7 +70,10 @@ in
           ++ layers;
 
         maxLayers = 25;
-        copyToRoot = [setupLinks] ++ setup;
+        copyToRoot = nixpkgs.buildEnv {
```

Comment:
What benefit is this bringing?

Reply:
without this, any to files in `copyToRoot` will cause a conflict, even if its just a directory with the same name but different contents. `buildEnv` resolves these conflicts based on pakage priority, so unless two packages have the same exact priority, they won't conflict.

Also there is no conflict on directories, so they will just be merged together. Not only is this change not possible without this (for whatever reason it triggered a conflict), but it will make the root file system easier for the user to extend as they need to.

Reply:
I'm surprised I haven't run into this before 🤷 SGTM.